### ISSUE TITLE
run_wstool.sh fails with sudo: apt-add-repository: command not found

### ISSUE THREAD

**Issue by gilwoolee** [personalrobotics/pr-cleanroom#4]
`run_wstool.sh` fails with the following error:

```
Step 5 : RUN ./internal-setup.sh
 ---> Running in ed120c0c13dc
+ '[' '!' -f /etc/apt/sources.list.d/ros-latest.list ']'
+ sudo -n apt-add-repository -y ppa:libccd-debs
sudo: apt-add-repository: command not found
```

**Comment by mkoval** [comment]
I don't think this is related to `sudo`. It looks like you are missing the `apt-add-repository` command entirely. You may need to install:

``` console
$ apt-get install software-properties-common
```

**Comment by ClintLiddick** [comment]
@mkoval this is happening inside the docker container. I confirmed with calls to `docker run -it personalrobotics/ros-openrave bash -c "apt-add-repository --help"`. This may be related to https://github.com/personalrobotics/pr-cleanroom/pull/3 ?

**Comment by psigen** [comment]
The base image might no longer include the `add-apt-repository` command,
since that's part of `python-software-properties` which isn't required.

Can you try adding apt-get install `python-software-properties` first?
